magic-modules/merge-pr.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+echo "$CREDS" > ~/github_private_key
+chmod 400 ~/github_private_key
+
+
+shopt -s dotglob
+cd mm-approved-prs
```

Comment:
I'll need to sync w/ you to understand better how these folders go in an out. And why not share them between jobs using volumes. Less copy?

Reply:
Concourse!  Concourse has inputs and outputs to each job, and inputs and outputs are both just folders.  In this case, they're git repositories.  You can't have an input be the same folder as an output, so unfortunately we're stuck with this copying.

Reply:
OK.